crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -111,6 +112,7 @@ impl From<UserIdentity> for UserIdentities {
 pub struct OwnUserIdentity {
     pub(crate) inner: ReadOnlyOwnUserIdentity,
     pub(crate) verification_machine: VerificationMachine,
+    store: Store,
```

Comment:
Do we need this? The `VerificationMachine` already has access to a store type, can't we get things from there?

I guess the point here is that the store inside `VerificationMachine` doesn't have a `ReadOnlyAccount` anymore?

Reply:
> Do we need this? The VerificationMachine already has access to a store type, can't we get things from there?

Yes. No, the "store" in the `VerificationStore` contained in the `VerificationMachine` was containing the `ReadOnlyAccount` before, and now it's only containing a `StaticAccountData` instead.

> I guess the point here is that the store inside VerificationMachine doesn't have a ReadOnlyAccount anymore?

So yeah, basically that.